python/graphstorm/inference/ep_infer.py

```diff
@@ -69,7 +69,7 @@ def infer(self, loader, save_embed_path, save_prediction_path=None,
         do_eval = self.evaluator is not None
         sys_tracker.check('start inferencing')
         self._model.eval()
-        embs = do_full_graph_inference(self._model, loader.data,
+        embs = do_full_graph_inference(self._model, loader.data, fanout=loader._fanout,
```

Comment:
you still need to expose `fanout` as a property

Reply:
Done.